react/features/filmstrip/middleware.web.js

```diff
@@ -210,7 +210,8 @@ MiddlewareRegistry.register(store => next => action => {
             break;
         }
 
-        if (stageFilmstrip && currentLayout === LAYOUTS.VERTICAL_FILMSTRIP_VIEW) {
+        if (stageFilmstrip && (currentLayout === LAYOUTS.VERTICAL_FILMSTRIP_VIEW
```

Comment:
Do we have an use case where `stageFilmstrip` will be true and the `currentLayout` will be the vertical filmstrip? Also I'm wondering if it would appropriate to use `shouldDisplayStageFilmstrip` here instead of checking for stageFilmstrip and the currentLayout?

Reply:
Changed it to use `shouldDisplayStageFilmstrip`